libraries/AP_HAL/board/linux.h

```diff
@@ -387,9 +351,6 @@
 #define HAL_GPS_TYPE_DEFAULT 9
 #define HAL_CAN_DRIVER_DEFAULT 1
 #elif CONFIG_HAL_BOARD_SUBTYPE == HAL_BOARD_SUBTYPE_LINUX_RST_ZYNQ
-#define HAL_BOARD_LOG_DIRECTORY "/mnt/APM/logs"
-#define HAL_BOARD_TERRAIN_DIRECTORY "/mnt/APM/terrain"
-#define HAL_BOARD_STORAGE_DIRECTORY "/mnt/APM"
```

Comment:
This board wasn't using the default...

Reply:
#7100

And the ones that are not related to Android or a custom distro seem to use the old default just because it was the default, aka copy and paste effect.  Here I prefer to make the new default global and revert for whom complains.

Reply:
I'm adding more information on the commit message and will ping board maintainers

Reply:
If you are going to ping board maintainers, that's fine, otherwise we can't just change it, it might break their boards.